sinks/signalfx/signalfx.go

```diff
@@ -51,6 +51,7 @@ func (c *collection) submit(ctx context.Context, cl *trace.Client) error {
 		err := client.AddDatapoints(ctx, points)
 		if err != nil {
 			span.Error(err)
+			span.Add(ssf.Count("flush.error_total", 1, map[string]string{"cause": "io"}))
```

Comment:
Hrmmm, if you can't tell them apart, should we maybe add the sink name to the dd sink and this one too?

Reply:
Also, since we're starting to duplicate that name, maybe we can make it a constant on the sinks package? (Optional!)

Reply:
yeah i was thinking we thread the sink name down into the PostHelper somehow, i'll take a closer look

re the second comment - tricky, because the posthelper composes the metric name at runtime (`action+".error_total"`)

Reply:
OK on the second thing - let's just synthesize the right value then (maybe leave a comment on PostHelper that we're also generating one of those elsewhere though?)

Reply:
yeah i'll leave a comment on this callsite